az-digital/az_quickstart
# Closes #3014, Closes #3017: Update roles with additional block content permissions added to Drupal core 10.1 and deprecate block_content_permissions module.

## Description
- Updates permissions in roles based on changes to block content permissions in Drupal 10.1 (see [core change record](https://www.drupal.org/node/3346838))
- Deprecates block_content_permissions contrib module (includes changes from #3020)
- Adds DB update to remove no-longer-needed block content permissions from AZQS roles

## Related issues




Closes #3014
Closes #3017 

## How to test



- We need to ensure content admins and content editors can still perform all of the block_content related tasks they need to
- We need to ensure that the DB update included in this PR works and that the updated roles don't still have a module dependency on the `block_content_permissions` module (ensure that the module can be uninstalled without causing roles to get deleted or anything)

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
